File: openapi-client/javascript-closure-angular/API/Client/SendMailAdvCcInner.js
goog.provide('API.Client.SendMailAdv_cc_inner');

/**
 * An Email Contact
 * @record
 */
API.Client.SendMailAdvCcInner = function() {}

/**
 * The email address
 * @type {!string}
 * @export
 */
API.Client.SendMailAdvCcInner.prototype.email;

/**
 * (optional) Name to use for the CC contact.
 * @type {!string}
 * @export
 */
API.Client.SendMailAdvCcInner.prototype.name;

File: openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/SendMailAdvCcInner.scala

package org.openapitools.client.model


case class SendMailAdvCcInner (
    /* The email address */
    _email: String,
    /* (optional) Name to use for the CC contact. */
    _name: Option[String]
)
object SendMailAdvCcInner {
    def toStringBody(var_email: Object, var_name: Object) =
        s"""
        | {
        | "email":$var_email,"name":$var_name
        | }
        """.stripMargin
}
